packages/strapi-parts/src/Layout/HeaderLayout.js

```diff
@@ -1,72 +1,131 @@
-import React from 'react';
+import React, { useState, useRef, useEffect } from 'react';
+import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import { H1, Subtitle, P, H2 } from '../Text';
 import { Box } from '../Box';
 import { Row } from '../Row';
 
-export const HeaderLayout = ({
-  navigationAction,
-  primaryAction,
-  secondaryAction,
-  subtitle,
-  title,
-  sticky,
-  ...props
-}) => {
-  if (sticky) {
+const useElementOnScreen = (options) => {
+  const containerRef = useRef(null);
+  const [isVisible, setIsVisible] = useState(true);
+
+  const cb = ([entry]) => {
+    setIsVisible(entry.isIntersecting);
+  };
+
+  useEffect(() => {
+    const observer = new IntersectionObserver(cb, options);
+    if (containerRef.current) observer.observe(containerRef.current);
+
+    return () => {
+      if (containerRef.current) observer.unobserve(containerRef.current);
+    };
+  }, [containerRef, options]);
+
+  return [containerRef, isVisible];
+};
+
+export const HeaderLayout = (props) => {
+  const [containerRef, isVisible] = useElementOnScreen({
+    root: null,
+    rootMargin: '0px',
+    threshold: 0,
+  });
+
+  return (
+    <>
+      <BaseHeaderLayout ref={containerRef} {...props} />
+      {!isVisible && <BaseHeaderLayout {...props} sticky />}
```

Comment:
Do you think it would be possible to hide the `BaseHeaderLayout` that is not sticky, somehow to avoid the following scenario when reverse tabbing?



![2021-08-13 10 32 45](https://user-images.githubusercontent.com/3874873/129328961-12c864f7-0ffd-43cf-a567-4e5ab7d9419d.gif)

Because it also duplicate the entries for assistive technologies 😅 

![2021-08-13 10 34 42](https://user-images.githubusercontent.com/3874873/129329150-748ce390-01eb-4b2c-8b34-283e61e1ce50.gif)

Reply:
Also, what if the HeaderLayout is an h1? I think it would duplicate it right? It's not possible to do that in semantic HTML :(